packages/wallets/src/components/WalletsContainer/WalletsContainer.tsx

```diff
@@ -22,7 +23,7 @@ const WalletsContainer: React.FC<React.PropsWithChildren<TProps>> = ({ children,
             }
         }, 300);
         return () => clearTimeout(timeout);
-    }, [isOpen]);
+    }, [isOpen, defaultLanguage]);
```

Comment:
Added defaultLanguage as dependency to scroll page back to top when language is switched

https://github.com/user-attachments/assets/68f6874b-c9da-40b8-a258-f6ae654f5583